pact/pact.py

```diff
@@ -103,26 +100,20 @@ def given(self, provider_state):
         :type provider_state: basestring
         :rtype: Pact
         """
-        self._provider_state = provider_state
+        self._interactions.insert(0, {'provider_state': provider_state})
```

Comment:
Heh. No, there wasn't. It is just an artifact of my thought process... The list was, in my head, a stack that I was pushing/peeking.